code/modules/clothing/suits/ego_gear/teth.dm

```diff
@@ -147,7 +147,14 @@
 	armor = list(RED_DAMAGE = 10, WHITE_DAMAGE = 20, BLACK_DAMAGE = -10, PALE_DAMAGE = 0) //20
 
 /obj/item/clothing/suit/armor/ego_gear/curfew
-	name = "Curfew"
+	name = "curfew"
 	desc = "Everything was rotting but the memory of that night wasn't."
 	icon_state = "curfew"
 	armor = list(RED_DAMAGE = -20, WHITE_DAMAGE = 20, BLACK_DAMAGE = 0, PALE_DAMAGE = 20) // 20
+
+/obj/item/clothing/suit/armor/ego_gear/page
+	name = "page"
+	desc = "When one has filled book in whatever way, then the book may grant one wish. \
+	It may sound absurd but many take the rumor seriously."
+	icon_state = "page"
+	armor = list(RED_DAMAGE = -20, WHITE_DAMAGE = 10, BLACK_DAMAGE = 30, PALE_DAMAGE = 0) // 20
```

Comment:
Pretty sure this already exists (but I'm in bed and can't check the spreadsheet rn)
-1/1/2/0, -1/0/3/0, and any and all variations on that are available

Reply:
This spread is the only one of its kind.